File: app/app/src/main/java/me/williamhester/reddit/ui/fragments/SubmissionFragment.java
package me.williamhester.reddit.ui.fragments;

import android.os.Bundle;

/** A fragment that shows Submissions */
public class SubmissionFragment extends VotableFragment {

  public static SubmissionFragment newInstance() {
    Bundle args = new Bundle();
    SubmissionFragment fragment = new SubmissionFragment();
    fragment.setArguments(args);
    return fragment;
  }

  @Override
  protected void loadContent() {
    redditClient.getSubmissions("", null, getAfter());
  }
}

File: app/app/src/main/java/me/williamhester/reddit/ui/fragments/SubmissionFragment.kt
package me.williamhester.reddit.ui.fragments

import android.os.Bundle

/** A fragment that shows Submissions  */
class SubmissionFragment : VotableFragment() {

  override fun loadContent() {
    redditClient.getSubmissions("", null, after)
  }

  companion object {
    fun newInstance(): SubmissionFragment {
      val args = Bundle()
      val fragment = SubmissionFragment()
      fragment.arguments = args
      return fragment
    }
  }
}
